.gitignore

```diff
@@ -0,0 +1 @@
+*.log
```

Comment:
I was learning more about version control and git recently, I think it's great to have gitignore file to untrack some files. Should we also ignore some .DS_Store file too? Could you also have some comment here to show what is .log file and why should we ignore them?